benchmarks/dynamo/timm_models.py

```diff
@@ -205,8 +205,7 @@ def load_model(
                     drop_rate=0.0,
                     drop_path_rate=None,
                     drop_block_rate=None,
-                    # Skip downloading pretrained models for speedy CI
-                    pretrained=not self.args.ci,
+                    pretrained=True,
```

Comment:
@malfet , this change should eliminate the flakiness we have seen. I will keep an eye on HUD after landing this.